tests/cdb2_coltype.test/test.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
```

Comment:
Could you use tests/tools/compare_results.sh instead?

Reply:
Yes, I'll try to do that.